### PR TITLE
Compile in setup.py

### PR DESCRIPTION
**Pull request by jburel**
Compile code in setup.py

cc @aleksandra-tarkowska @waxenegger

### CODE REVIEW

**Comment by jburel** [comment]
``python setup.py sdist upload -r https://testpypi.python.org/pypi`` works fine 
see https://testpypi.python.org/pypi?:action=display&name=omero-iviewer&version=0.0.2

File: plugin/setup.py [hunk 1]
```diff
@@ -35,6 +38,46 @@ def read(fname):
...
+class RunProd(Command):
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        for command in self.get_sub_commands():
```
Comment (aleksandra-tarkowska) [inline plugin/setup.py]: if you are not listing subcommands that can be removed

**Comment by aleksandra-tarkowska** [comment]
I am afraid you have to distinguish in `run` if src is available or not, in figure it is checking for `if not os.path.isdir('src'):` https://github.com/ome/omero-figure/pull/188/commits/6bcf513dda5c07240c864dbd42e551d6ee5b052f#diff-2eeaed663bd0d25b7e608891384b7298R100. Otherwise it will try to compile even pre-compiled package

**Comment by jburel** [comment]
oops I removed those lines by mistake

File: plugin/setup.py (hunk 1 above)
Comment (jburel) [inline plugin/setup.py]: Was planning to go back to it today, I had enough last night ;-)

**Comment by aleksandra-tarkowska** [comment]
I am affraid this is still not wrking https://ci.openmicroscopy.org/view/Failing/job/WEB-DEV-trial-deploy/112/console

**Comment by jburel** [comment]
This is due to the fact that the command in job is looking for setup.py at the level of omero-iviewer and not omero-iviewer/plugin where it is currently

**Comment by aleksandra-tarkowska** [comment]
then you need to pip install using `subdirectory`

**Comment by jburel** [comment]
will look into that later

**Comment by jburel** [comment]
https://ci.openmicroscopy.org/view/Failing/job/WEB-DEV-trial-deploy/114/console error
Will check post release

**Comment by aleksandra-tarkowska** [comment]
job is green now

**Comment by jburel** [comment]
Merging so @waxenegger can move forward

**Pull request merged**